src/main/java/net/imagej/ops/image/ImageNamespace.java

```diff
@@ -456,6 +457,80 @@
 		return result;
 	}
 
+	// -- quality --
+
+	@OpMethod(op = net.imagej.ops.image.quality.DefaultMAE.class)
+	public <T extends RealType<T>> DoubleType mae(final DoubleType out,
+		final IterableInterval<T> reference, final IterableInterval<T> test)
+	{
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.image.quality.DefaultMAE.class, out, reference, test);
+		return result;
+	}
+
+	@OpMethod(op = net.imagej.ops.image.quality.DefaultMAE.class)
+	public <T extends RealType<T>> DoubleType mae(
+		final IterableInterval<T> reference, final IterableInterval<T> test)
+	{
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.image.quality.DefaultMAE.class, reference, test);
+		return result;
+	}
+
+	@OpMethod(op = net.imagej.ops.image.quality.DefaultPSNR.class)
+	public <T extends RealType<T>> DoubleType psnr(final DoubleType out,
+		final IterableInterval<T> reference, final IterableInterval<T> test)
+	{
+		final DoubleType result = (DoubleType) ops().run(
+			net.imagej.ops.image.quality.DefaultPSNR.class, out, reference, test);
```

Comment:
Should be `ops().run(Ops.Image.PSNR.class, out, reference, test)`, and same idea for the remaining namespace methods you added. They should be `run(...)` with an interface not a concrete implementation.